docs/book/unit-testing.md

```diff
@@ -27,11 +27,16 @@ integration for laminas-mvc, including application scaffolding and custom
 assertions. You will need to install it:
 
 ```bash
-$ composer require --dev laminas/laminas-test phpunit/phpunit
+$ composer require --dev laminas/laminas-test phpunit/phpunit:<version>
 ```
 
-laminas-test package supports very wide range of PHPUnit versions, make sure to
-always explicitly require phpunit/phpunit versions that are compatible with your tests.
+where `<version>` is a version of `phpunit/phpunit` that is compatible with your tests.
+`laminas-test` package supports very wide range of PHPUnit versions, make sure to
+always explicitly require phpunit/phpunit versions that are compatible with your tests. 
+If you do not explicitely require a phpunit/phpunit version, the latest version supported by laminas-test
+will be installed.
```

Comment:
I don't know if we really need the part with the versions in this introduction – I mean both the new and the old sentence. 🤔 


There are duplications:

> …that is compatible with your tests.
> …
> …that are compatible with your tests.

Reply:
To get the user going quickly and since phpunit is not installed with the skeleton, and since laminas-test requires phpunit, it will be installed. So I took out the version part. 
I kept a mention that phpunit will be installed.